README.adoc

```diff
@@ -45,6 +45,10 @@ public static HoverflyRule hoverflyRule = HoverflyRule.inCaptureMode(classpath("
 // After the capturing, switch to inSimulationMode to spin up a stub server
 @ClassRule
 public static HoverflyRule hoverflyRule = HoverflyRule.inSimulationMode(classpath("simulation.json"));
+
+// Or you can use both approaches at once. If json file not present in capture mode, if present in simulation mode
+@ClassRule
+public static HoverflyRule hoverflyRule = HoverflyRule.inCaptureOrSimulationMode("simulation.json");
```

Comment:
What do you and @mogronalol think about this as the name?
``` 
HoverflyRule.newInferredRule("simulation.json");
```
It is not a new mode provided by hoverfly, so maybe we stick with the `newInstance` naming convention for static factory method.